usr/src/uts/common/fs/zfs/dmu_send.c

```diff
@@ -392,6 +392,22 @@ static int
 dump_freeobjects(dmu_sendarg_t *dsp, uint64_t firstobj, uint64_t numobjs)
 {
 	struct drr_freeobjects *drrfo = &(dsp->dsa_drr->drr_u.drr_freeobjects);
+	uint64_t maxobj = DNODES_PER_BLOCK *
+	    (DMU_META_DNODE(dsp->dsa_os)->dn_maxblkid + 1);
+
+	/*
```

Comment:
Shouldn't this comment be updated for openzfs, describing which openzfs "versions" are affected?

Reply:
You're right, but how do I find out the version number? Do you use the git commit hash as version number? Or is there somewhere I can find that information?

Reply:
A date might be the only useful way to express this

Reply:
Yes, we should update the comment.  The simplest change would be to say `Older versions of ZFS did not handle ...`.  If we want to be more specific, we could say `Prior to the fix for bug 1234, ZFS did not handle...`